### DISCUSSION TITLE
Using Developer Certificate of Origin (DCO) for code sign-off

### DISCUSSION THREAD

**Discussion by dennyglee** [delta-io/delta-rs#2732] [Ideas]
As Delta Lake is part of the LF AI & Data, the recommended approach for code sign-off is a Developer Certificate of Origin (DCO).   For more information, please refer to this [draft proposal](https://docs.google.com/document/d/1PIY_IgRQ-i1ZtQS0WbnKVcmi52PpoN_AqAs3ZsjGnag/edit).  

By definition, our [CONTRIBUTING.md](https://github.com/delta-io/delta-rs/blob/main/CONTRIBUTING.md) follows the DCO approach.  Though, we'd like to recommend:
* Clarifying this similar to how [Chef DCO](https://github.com/chef-boneyard/chef-rfc/blob/76778b8c5b51444f973a0daecb5de471dc9ec134/rfc080-dco.md#L4) (here is a [blog](https://www.chef.io/blog/introducing-developer-certificate-of-origin) on this topic)
* Enforcing DCO to ensure that contributions are submitted by the author to this project.

Please feel free to discuss if you have any concerns with this approach.

**Comment by rtyler** [comment]
Makes sense to me :+1:

**Comment by rtyler** [comment]
DCO has been enabled on this repository now :partying_face:
